### PR TITLE
chore(vendor): update go-common-libs/go-iscsi-utils and resolve break…

### PR DESCRIPTION
**Pull request by ChanYiLin**
ref: https://github.com/longhorn/longhorn/issues/7960

Update dependencis and resolve the breaking changes.
- go-iscsi-helper
- go-common-libs
The PR is mainly based on this PR: https://github.com/longhorn/longhorn-share-manager/pull/80/files

### CODE REVIEW

**Comment by ChanYiLin** [comment]
**CORE TEST SUCCEEDED**
https://ci.longhorn.io/job/private/job/longhorn-tests-regression/6623/console

File: pkg/crypto/crypto.go [hunk 1]
```diff
@@ -26,8 +36,14 @@ func OpenVolume(volume, devicePath, passphrase string) error {
...
 
+	namespaces := []lhtypes.Namespace{lhtypes.NamespaceMnt, lhtypes.NamespaceIpc}
+	nsexec, err := lhns.NewNamespaceExecutor(lhtypes.ProcessNone, lhtypes.HostProcDirectory, namespaces)
+	if err != nil {
+		return err
+	}
+
 	logrus.Debugf("Opening device %s with LUKS on %s", devicePath, volume)
-	_, err := luksOpen(volume, devicePath, passphrase)
+	_, err = nsexec.LuksOpen(volume, devicePath, passphrase, lhtypes.LuksTimeout)
```
Comment (derekbit) [inline pkg/crypto/crypto.go:46]: We don't log the stdout and, especially, stderr of LuksOpen(). Can you help create a ticket for improvement? Thank you.

**Review by derekbit — APPROVED** [review]
LGTM

File: pkg/crypto/crypto.go (hunk 1 above)
Comment (ChanYiLin) [reply pkg/crypto/crypto.go:46]: Sure, created
https://github.com/longhorn/longhorn/issues/8171

**Review by innobead — APPROVED** [review]

**Pull request merged**